beacon-chain/rpc/validator/exit.go

```diff
@@ -0,0 +1,75 @@
+package validator
+
+import (
+	"context"
+
+	ptypes "github.com/gogo/protobuf/types"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/statefeed"
+	"github.com/prysmaticlabs/prysm/shared/bls"
+	"github.com/prysmaticlabs/prysm/shared/mathutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/roughtime"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// RequestExit requests an exit for a validator.
+func (vs *Server) RequestExit(ctx context.Context, req *ethpb.VoluntaryExit) (*ptypes.Empty, error) {
```

Comment:
`RequestExit` is a bit confusing. It sounds like validator is requesting the exit object from the beacon node. From the conventions in `attester` and `proposer`,  I think we should call this `SubmitExit` or `ProposeExit`...

Reply:
`RequestExit` is the name in the RPC call so I'd have to change it there first.  Should I still go ahead and alter it?  If so I'd go for `ProposeExit` as `SubmitExit` sounds a bit like it's already happened.

Reply:
Let's change the upstream definitions. `ProposeExit` sounds good to me!

Reply:
Cool; will sort that and update this PR when the update to ethereumapis is in place.

Reply:
Renamed to `ProposeExit` in latest commit.